### ISSUE TITLE
errored rendering of <u> elements in PDF

### ISSUE THREAD

**Issue by alicerusso** [ietf-tools/xml2rfc#1192]
### Describe the issue

scrambled rendering of <u> elements in the PDF. (HTML and TXT are ok.)  see 
A) https://www.rfc-editor.org/v3test/test9639_A.png (scrambled order and text is missing; this is from p. 62 of the PDF below) vs.
B) https://www.rfc-editor.org/v3test/test9639_B.png (correct order. only change was to text in <t> from "as follows:" to "xx xxxxxxx:")

Line break location relative to RTL character in the PDF seems to be a factor.

files:
https://www.rfc-editor.org/v3test/test9639v8.xml
https://www.rfc-editor.org/v3test/test9639v8.pdf (page 62)

### Code of Conduct

- [x] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by cabo** [comment]
Workaround: Avoid having a line break right before the "ש":

<img width="538" alt="Image" src="https://github.com/user-attachments/assets/f6a2792b-5bbc-4ef6-9ebb-89c7d7d571e7" />

**Comment by alicerusso** [comment]
Right; a workaround is shown in B above. (A workaround was used, and the document was published.)
